### PR TITLE
qa_crowbarsetup: Update after change of nova roles names

### PR DESCRIPTION
**Pull request by vuntz**
See https://github.com/crowbar/crowbar-openstack/pull/68

### CODE REVIEW

**Comment by rsalevsky** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2749,7 +2759,7 @@ function get_novacontroller()
     local element=`crowbar nova proposal show default | \
         rubyjsonparse "
                     puts j['deployment']['nova']\
-                        ['elements']['nova-multi-controller']"`
+                        ['elements']['`nova_role_prefix`-controller']"`
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: :-1: test failed with qa_crowbarsetup.sh: command substitution: line 2769: unexpected EOF while looking for matching `"'

Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: Fixed.

**Comment by vuntz** [comment]
As mentioned in https://github.com/crowbar/crowbar-openstack/pull/68, a build with this and https://github.com/SUSE-Cloud/cct/pull/39 works fine.

**Comment by vuntz** [comment]
Argh, pressed enter too fast: anyone else with a spare +1/-1? :-)

**Comment by aplanas** [comment]
+1

**Pull request merged**